packages/cli/src/commands/certs/command.ts

```diff
@@ -82,16 +82,18 @@ export const certsCommand: Command = {
         'Number of results to return per page (default: 20, max: 100)',
       argument: 'VALUE',
       shorthand: null,
-      type: String,
+      type: Number,
       deprecated: false,
     },
     {
       name: 'next',
       description: 'Show next page of results',
-      shorthand: 'n',
-      type: String,
+      shorthand: 'N',
+      type: Number,
       deprecated: false,
     },
+    { name: 'overwrite', shorthand: null, type: Boolean, deprecated: false },
+    { name: 'output', shorthand: null, type: String, deprecated: false },
```

Comment:
Do we want to add a description and show help text for these flags? We've been parsing them but not showing them in the help until now.